mmdet/utils/replace_cfg_vals.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+import re
+
+from mmcv.utils import Config
+
+
+def replace_cfg_vals(ori_cfg):
```

Comment:
Please give the detailed example in PR message, like  https://github.com/open-mmlab/mmdetection/pull/7740